test/runtests.jl

```diff
@@ -2,11 +2,14 @@ using GasModels
 
 import InfrastructureModels
 import Memento
+import Logging
 
 # Suppress warnings during testing.
 const TESTLOG = Memento.getlogger(GasModels)
 Memento.setlevel!(TESTLOG, "error")
 Memento.setlevel!(Memento.getlogger(InfrastructureModels), "error")
+Logging.disable_logging(Logging.Info)
+Logging.disable_logging(Logging.Warn)
```

Comment:
Just curious, given that we use Memento for logging, wondering why this is here as well.

Reply:
Juniper appears to use `Logging` instead of `Memento`. This is mostly for disabling Juniper info and warning output.